src/platforms/android/migration.mdx

```diff
@@ -333,7 +412,7 @@ Sentry.configureScope { it.clear() }
 
 ### Sentry Gradle Plugin
 
-Disable the `autoProguardConfig` flag from the Sentry Gradle Plugin, `sentry-android` >= v2.0 already does it automatically.
+Disable the `autoProguardConfig` flag from the Sentry Gradle Plugin, `io.sentry:sentry-android` >= `2.0.0` already does it automatically.
```

Comment:
```suggestion
Disable the `autoProguardConfig` flag from the Sentry Gradle Plugin since `io.sentry:sentry-android` >= `2.0.0 does this automatically.
```